proto/multistaking/v1/query.proto

```diff
@@ -0,0 +1,102 @@
+syntax = "proto3";
+package multistaking.v1;
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+import "multistaking/v1/multi_staking.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "google/api/annotations.proto";
+
+option go_package = "github.com/realiotech/multi-staking/x/multi-staking/types";
+
+// Query defines the gRPC querier service.
+service Query {
+  rpc MultiStakingLocks(QueryMultiStakingLocks)
+      returns (QueryMultiStakingLocksResponse) {
+    option (google.api.http).get = "/realiotech/multistaking/v1/locks";
+  }
+  rpc MultiStakingLock(QueryMultiStakingLock)
+      returns (QueryMultiStakingLockResponse) {
+    option (google.api.http).get =
+        "/realiotech/multistaking/v1/lock/{validator_addr}/{delegator_addr}";
+  }
+  rpc MultiStakingUnlocks(QueryMultiStakingUnlocks)
+      returns (QueryMultiStakingUnlocksResponse) {
+    option (google.api.http).get = "/realiotech/multistaking/v1/unlocks";
+  }
+  rpc MultiStakingUnlock(QueryMultiStakingUnlock)
+      returns (QueryMultiStakingUnlockResponse) {
+    option (google.api.http).get =
+        "/realiotech/multistaking/v1/unlock/{validator_addr}/{delegator_addr}";
+  }
+  rpc ValidatorMultiStakingCoin(QueryValidatorMultiStakingCoin)
+      returns (QueryValidatorMultiStakingCoinResponse) {
+    option (google.api.http).get =
+        "/realiotech/multistaking/v1/validator-coin/{validator_addr}";
+  }
+  rpc BondWeight(QueryBondWeight) returns (QueryBondWeightResponse) {
+    option (google.api.http).get = "/realiotech/multistaking/v1/weight/{denom}";
+  }
+}
+
+message QueryMultiStakingLocks {
```

Comment:
we should do query via ID

Reply:
We should use string for easy developing at FE side, an url shouldn't have struct inside itself.

Reply:
then it should be multistaker_addr and val_addr

Reply:
@neitdung

Reply:
ohh sorry I misread this, it's query locks not a single lock